concepticondata/conceptlists/Lundgren-2020-275.tsv

```diff
@@ -0,0 +1,276 @@
+ID	NUMBER	NO	ENGLISH	CONCEPTICON_ID	CONCEPTICON_GLOSS
+Lundgren-2020-275-1	1	1	ACHIOTE	349	ACHIOTE
+Lundgren-2020-275-2	2	2	AFTER	1685	AFTER
+Lundgren-2020-275-3	3	3	AFTERNOON	1807	AFTERNOON
+Lundgren-2020-275-4	4	4	AGOUTI	313	AGOUTI
+Lundgren-2020-275-5	5	5	AJÍ		
+Lundgren-2020-275-6	6	6	ALL	98	ALL
+Lundgren-2020-275-7	7	7	ALSO~ALREADY		
+Lundgren-2020-275-8	8	8	ANGRY	1826	ANGRY
+Lundgren-2020-275-9	9	9	ANT	587	ANT
+Lundgren-2020-275-10	10	10	ANTEATER	181	ANTEATER
+Lundgren-2020-275-11	11	11	ARM	1673	ARM
+Lundgren-2020-275-12	12	12	ARMADILLO	1885	ARMADILLO
+Lundgren-2020-275-13	13	13	ARRIVE (INTR.)	1387	ARRIVE
+Lundgren-2020-275-14	14	14	ARROW	977	ARROW
+Lundgren-2020-275-15	15	15	ASHES	646	ASH
+Lundgren-2020-275-16	16	16	ATTACH		
+Lundgren-2020-275-17	17	17	AUNT	1272	AUNT
+Lundgren-2020-275-18	18	18	AXE	677	AXE
+Lundgren-2020-275-19	19	19	BACK		
+Lundgren-2020-275-20	20	20	BAD	1292	BAD
+Lundgren-2020-275-21	21	21	BAT	1793	BAT
+Lundgren-2020-275-22	22	22	BATHE (INTR.)	138	BATHE
+Lundgren-2020-275-23	23	23	BE BORN (INTR.)	1090	BE BORN
+Lundgren-2020-275-24	24	24	BE DRY	1398	DRY
+Lundgren-2020-275-25	25	25	BE FAT		
```

Comment:
1279 FAT (OBESE)?

Reply:
I left this one unmapped, because it's unclear if the concept refers to the description of a human body (and in Kokama it is) or to meat/oil. I can't find more information on the meaning of the corresponding lexemes in the other languages.